packages/rangelink-vscode-extension/src/__integration-tests__/suite/editorBindingValidation.test.ts

```
@@ -13,13 +11,15 @@ import {
   findTestItemsByPrefix,
   getLogCapture,
   getWorkspaceRoot,
+  openAndDismiss,
   settle,
   standardSuite,
-  waitForHuman,
   waitForHumanVerdict,
 } from '../helpers';
 
-standardSuite('Editor Binding Validation', { assisted: true }, (log) => {
+import assert from 'node:assert';
```

ESLint

[error] `node:assert` import should occur before import of `node:fs`

(import/order)